rskj-core/src/main/java/co/rsk/cli/tools/ShowInfo.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2021 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.cli.tools;
+
+import co.rsk.RskContext;
+import co.rsk.cli.CliToolRskContextAware;
+import co.rsk.trie.Trie;
+import co.rsk.trie.TrieStore;
+import org.ethereum.core.Block;
+import org.ethereum.db.BlockStore;
+import org.spongycastle.util.encoders.Hex;
+
+import javax.annotation.Nonnull;
+import java.lang.invoke.MethodHandles;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * The entry point for export state CLI tool
+ * This is an experimental/unsupported tool
+ *
+ * This tool can be interrupted and re-started and it will continue the migration from
+ * the point where it left.
+ *
+ * Required cli args:
+ *
+ * - args[0] - command "stateroot" / blocknumber / dbFormat / dbPath /triestorePath
+ *
+ * STATEROOT:
+ * - args[1] - block number
+
+ *
+ */
+public class ShowInfo extends CliToolRskContextAware  {
+    private static final int COMMAND_IDX = 0;
+    private static final int BLOCK_NUMBER_IDX = 1;
+
+    public static void main(String[] args) {
+        create(MethodHandles.lookup().lookupClass()).execute(args);
```

Comment:
we could make use of `picocli`, same as in the `MigrateState` tool

Reply:
Addressed